src/goose/defaults.clj

```diff
@@ -59,7 +71,7 @@
 (def rmq-default-url "amqp://guest:guest@localhost:5672")
 (def rmq-producer-channels 5)
 (def rmq-prefetch-limit 1)
-(def rmq-delay-limit-ms 4294967295) ; (2^32 - 1)
+(def rmq-delay-limit-ms 4294967295)                         ; (2^32 - 1)
```

Comment:
Was this change unintentional?

Reply:
This was an auto formatting done by my IDE, which aligns all inline comments to column 60. 

Should this be reverted?

Reply:
Please revert this, and change IDE setting to ignore comment alignment.